Docs/topics/vita.md

```diff
@@ -0,0 +1,92 @@
+# Using Allefresher Vita to connect LBP Vita to custom servers
+
+<include from="Library.topic" element-id="supported-version-notice"/>
+
+Allefresher Vita is a CFW plugin which patches LBP Vita to custom servers. This guide aims to show how to set it up on a modded Vita.
+
+To download Allefresher, you must first visit the [Releases Page](https://github.com/LittleBigRefresh/Allefresher/releases/latest). 
+Once you have opened it in your browser, select `Allefresher_vita.suprx`
+
+![Allefresher Artifacts](allefresher-artifacts.png)
+
+Once you have downloaded it, keep it on hand for later.
+
+## Step 1: Jailbreak your device and install a CFW
+
+We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guide on [vita.hacks.guide](https://vita.hacks.guide/). 
+You should install the HENkaku Ensō CFW, as that is what we officially support.
+
+After you have jailbroken your Vita, install [VitaShell](https://github.com/TheOfficialFloW/VitaShell/releases/latest).
+
+## Step 2: Start and connect to an FTP server on your Vita
+
+In VitaShell, press "Start", and change the "SELECT button" option to FTP. Press start again and then press select. 
+This will tell you the IP you need to type into Windows Explorer's address bar. (e.g. `ftp://192.168.0.100:1337/`).
+After pressing enter, you will see the list of files on your Vita.
+
+![vita-windows-ftp.png](vita-windows-ftp.png)
+
+## Step 3: Install iTLS
+
+iTLS is a plugin which enables more modern encryption standards on Vita CFW. This is required for Allefresher Vita to function.
```

Comment:
```suggestion
iTLS is a plugin which enables TLS 1.2 (a more modern encryption standard) on Vita CFW. This is required for Allefresher Vita to function.
```

is it really required or do we just use it for lbr only

Reply:
it's *technically* not required for the plugin to work, but you need it for PSN on CFW aswell anyway, in 2021 PSN started requiring TLS1.2 with extra certs, and the CFW ver ensou is based on doesnt support TLS1.2